### PR TITLE
#24 Google calendar event

### PR DESCRIPTION
**Pull request by tantai1123**

### CODE REVIEW

File: .env_example [hunk 1]
```diff
@@ -23,3 +23,5 @@ REPLICA_URL=s3://<bucket>/<sub-folder>
 
 # cache configuration
 CACHE_REDIS_URL=
+
+MEETING_CALENDAR_ID=
```
Comment (huynq2007) [inline .env_example:27]: do not use .env for storing sensitive information

File: src/actions/action_create_meeting_event.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+    def _get_calendar_service():
+        """Shows basic usage of the Google Calendar API.
+        Prints the start and name of the next 10 events on the user's calendar.
+        """
+        creds = None
+        # The file token.json stores the user's access and refresh tokens, and is
+        # created automatically when the authorization flow completes for the first
+        # time.
+        if os.path.exists("token.json"):
```
Comment (huynq2007) [inline src/actions/action_create_meeting_event.py:68]: do not use `token.json` for authen as our app server doesn't support persist file system

**Review by huynq2007 — CHANGES_REQUESTED** [review]

**Comment by tantai1123** [comment]
@huynq2007 
I change to using the google service account
Please confirm!

File: src/actions/action_create_meeting_event.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            'start': {
+                'dateTime': event['startDatetime'],
+                'timeZone': 'Asia/Ho_Chi_Minh',
+            },
+            'end': {
+                'dateTime': event['endDatetime'] or event['startDatetime'],
+                'timeZone': 'Asia/Ho_Chi_Minh',
+            },
+        }
+        API_URL = self.get_config_value("API_URL")
```
Comment (huynq2007) [inline src/actions/action_create_meeting_event.py:57]: remove used variable 
API_URL = self.get_config_value("API_URL")
@tantai1123

File: src/config.py [hunk 4]
```diff
@@ -35,3 +36,4 @@ class Config:
     CACHE_TYPE = "RedisCache"
     CACHE_DEFAULT_TIMEOUT = 1800
     CACHE_REDIS_URL = os.environ.get("CACHE_REDIS_URL") or "redis://redis:6379/0"
+    MEETING_CALENDAR_ID = os.environ.get("MEETING_CALENDAR_ID", "")
```
Comment (huynq2007) [inline src/config.py:39]: @tantai1123 
do not change things outside your action file

**Review by huynq2007 — APPROVED** [review]

**Pull request merged**